src/Feature/Changelog/Feature_Changelog.re

```diff
@@ -275,20 +281,36 @@ module View = {
         color(Colors.foreground.from(theme)),
       ];
 
-      let activeStyle = (font: UiFont.t, ~theme) => [
+      let linkActiveStyle = (font: UiFont.t, ~theme) => [
         fontFamily(font.fontFile),
         fontSize(12.),
         color(Colors.TextLink.activeForeground.from(theme)),
       ];
 
-      let inactiveStyle = (font: UiFont.t, ~theme) => [
+      let linkInactiveStyle = (font: UiFont.t, ~theme) => [
         fontFamily(font.fontFile),
         fontSize(12.),
         color(Colors.TextLink.foreground.from(theme)),
       ];
 
       let description = [marginVertical(6), flexDirection(`Row)];
     };
+
+    module Full = {
+      let titleActiveStyle = (font: UiFont.t, ~theme) => [
+        fontFamily(font.fontFile),
+        fontSize(12.),
+        color(
+          Colors.foreground.from(theme) |> Revery.Color.multiplyAlpha(0.75),
+        ),
+      ];
+
+      let titleInactiveStyle = (font: UiFont.t, ~theme) => [
+        fontFamily(font.fontFile),
+        fontSize(12.),
+        color(Colors.foreground.from(theme)),
+      ];
+    };
```

Comment:
The `Style` suffixes here are redundant, as they're already in a module called `Styles`.

Reply:
Very true, sorry about that! I had the tag name for the `ClickableText` elements on the brain which is why I messed that up. Thanks!

Reply:
Fixed in the latest commit!